### DISCUSSION TITLE
[Suggestion] Start (or resume) from a specific split

### DISCUSSION THREAD

**Discussion by rogeriotaques** [eplatonoff/pilorama#42] [Ideas]
After accidentally reset my counter I couldn't resume it. I had to start fresh. Think it would be nice to have a way to resume a paused counter or skip it until reaching the counter I want from my many splits.

**Comment by gadswan** [comment]
Hey @rogeriotaques, 

Thanks for the feedback, it's nice to know that someone is using the app and wants to help improve its UX

**Comment by rogeriotaques** [comment]
Hi, @gadswan! Thanks for the feedback.

I've been using `Be Focused` for a while and perhaps you guys can get some inspiration from them.
Anyway, you've done a good job on this so far. Keep it up! 👍
